Containers/mastercontainer/Caddyfile

```diff
@@ -10,6 +10,10 @@
     log {
         level ERROR
     }
+
+    servers {
+        protocols h1 h2
+    }
```

Comment:
```suggestion
    servers :80 {
        protocols h1 h2c
    }
```

Reply:
so that would only be disabled then for port 80? I want to disable it for all connections, see the reasoning above.

Reply:
but the caddy file only uses port 80?

Reply:
then at least also add h2c to this line

Reply:
and 8443, no?

Reply:
ohh yes

Reply:
I adjusted the suggestion